src/content/tutorials/user-auth-with-auth0/_index.md

```diff
@@ -0,0 +1,736 @@
+
+![Creating an application](/tutorials/user-auth-with-auth0/media/creating-an-application.png)
+
+Inside of your application's settings, the client ID and client secret are keys that we'll provide to our Workers application in order to authenticate with Auth0. There are a number of settings and configuration options here, but relevant to this tutorial are the "Allowed Callback URLs" and "Allowed Web Origins" options -- in the "Publish" section of this tutorial, we'll fill in these values with the final deployed URL of our application.
+
+## Generate a new project
+
+Using wrangler's `generate` command, we can begin building a new application using templates provided by the Workers team. For this tutorial, we'll heavily modify the default template for [Workers Sites](https://developers.cloudflare.com/workers/sites), which deploys a static HTML application:
+
+```sh
+$ wrangler generate --site my-auth-example
+```
+
+## Building an authorizer
+
+Before we begin implementing an "authorizer" in our application, which will verify that a user is logged in, it's useful to understand how Auth0's login flow works. The condensed version of this flow is below (and, if you're interested, you can [read a longer writeup in Auth0's docs](https://auth0.com/docs/flows/concepts/auth-code)):
+
+1. A user makes a request to the Workers application
+2. If they aren't logged in, they're redirected to the login page
+3. After logging in, they're redirected back to the Workers application, passing a login `code` as a query param
+4. The Workers application takes the login `code` parameter, and exchanges it with Auth0 for authorization tokens
+
+In a traditional application that is attached to a database, the authorization tokens returned from Auth0 are often persisted in a database, allowing users to return to the application and continue to use it, without the need for re-authorization. With a Workers application, we have a quick and easy-to-use data storage solution that lives right next to our serverless application: Workers KV. Using Workers KV, we'll store authorization tokens and tie them to a user using an authorization cookie.
+
+![Auth0 Flow](https://cdn2.auth0.com/docs/media/articles/flows/concepts/auth-sequence-auth-code.png)
+
+### Verifying a user
+
+Let's begin implementing the login flow described in the previous section. When a user makes a request to the Workers application, we should verify that the user is authenticated. To define this logic, create a new file, `workers-site/auth0.js`, which will contain the authorization/verification logic for our application:
```

Comment:
I think you can just stick with "authorizing" and "authenticating" rather than also introducing "verifying".  Here, you're `Authenticating a user`.